tests/Version.cpp

```diff
@@ -0,0 +1,111 @@
+#include "ziapi/Version.hpp"
+
+#include <gtest/gtest.h>
+
+TEST(Version, equal)
+{
+    ziapi::Version a{1, 0};
+    ziapi::Version b{1, 0};
+
+    ASSERT_TRUE(a == b);
+
+    a = ziapi::Version{1, 1};
+    b = ziapi::Version{1, 0};
+
+    ASSERT_FALSE(a == b);
+
+    a = ziapi::Version{2, 0};
+    b = ziapi::Version{1, 1};
+
+    ASSERT_FALSE(a == b);
+}
+
+TEST(Version, not_equal)
+{
+    ziapi::Version a{1, 0};
+    ziapi::Version b{1, 0};
+
+    ASSERT_FALSE(a != b);
+
+    a = ziapi::Version{1, 1};
+    b = ziapi::Version{1, 0};
+
+    ASSERT_TRUE(a != b);
+
+    a = ziapi::Version{2, 0};
+    b = ziapi::Version{1, 1};
+
+    ASSERT_TRUE(a != b);
+}
+
+TEST(Version, upper)
```

Comment:
```suggestion
TEST(Version, greater)
```
Just an english improvement